hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/util/WebAppUtils.java

```diff
@@ -509,11 +512,18 @@ static String getPassword(Configuration conf, String alias) {
       char[] passchars = conf.getPassword(alias);
       if (passchars != null) {
         password = new String(passchars);
+        LOG.info("Successful password retrieval for alias: {}", alias);
       }
     }
     catch (IOException ioe) {
       password = null;
+      LOG.error("Unable to retrieve password for alias: {}", alias, ioe);
```

Comment:
If i see well here the  `void error(String var1, Object var2, Object var3);` called, so the 3th throwable wont be logged in this case. Also in this case two error log will be created, so maybe debug level will be also fine here.

Reply:
Hello @K0K0V0K,
Thank you for your feedbacks. By default SLF4J logging will handle the Throwable as the last parameter for exception logging. Therefore, I do not need another place {} holder for it.

Reply:
I tried with a DEBUG level log and log message does not appear when I run the unit test that simulate the Exception. I think it better with an ERROR level because now I changed when the password is null it log with WARN level. So when the exception happens, it will log with two log messages in two levels ERROR and WARN.

Reply:
yup, seems you are right